x/avs/keeper/avs.go

```diff
@@ -2,22 +2,39 @@ package keeper
 
 import (
 	"fmt"
-
-	"github.com/ExocoreNetwork/exocore/x/avs/types"
+	"math/big"
+	"strings"
 
 	errorsmod "cosmossdk.io/errors"
 	sdkmath "cosmossdk.io/math"
 
+	"github.com/ExocoreNetwork/exocore/x/avs/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	ibcclienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/evmos/evmos/v14/x/evm/statedb"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func (k *Keeper) GetAVSSupportedAssets(ctx sdk.Context, avsAddr string) ([]string, error) {
+func (k *Keeper) GetAVSSupportedAssets(ctx sdk.Context, avsAddr string) (map[string]interface{}, error) {
 	avsInfo, err := k.GetAVSInfo(ctx, avsAddr)
 	if err != nil {
 		return nil, errorsmod.Wrap(err, fmt.Sprintf("GetAVSSupportedAssets: key is %s", avsAddr))
 	}
+	assetIDList := avsInfo.Info.AssetId
```

Comment:
Can you change the `Msg` definition in `protobuf` to say `AssetIDs` ? It gets confusing.
```
repeated string asset_ids = 5 [(gogoproto.customname) = "AssetIDs"];
```